### PR TITLE
feat(backup phrase): resize image on smaller height screens

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx [hunk 1]
```diff
@@ -34,6 +34,17 @@ const ButtonWrapper = styled(FlyoutWrapper)`
   }
 `
 
+const ImageContainer = styled.div`
+  @media only screen and (max-height: 600px) {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx:38]: This should not be allowed by style-lint

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx:38]: it didn't break nor stopped of committing, I can move it to service/ResponsiveService.

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx:38]: Yes, and please add @media to stylelint so we avoid this in the future.

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx]: never mind will update it here

File: package.json [hunk 2]
```diff
@@ -190,10 +190,10 @@
     "staged-git-files": "1.2.0",
     "standard": "12.0.1",
     "style-loader": "1.1.3",
-    "stylelint": "10.0.1",
-    "stylelint-config-recommended": "2.2.0",
+    "stylelint": "13.7.1",
+    "stylelint-config-recommended": "3.0.0",
     "stylelint-config-styled-components": "0.1.1",
-    "stylelint-processor-styled-components": "1.7.0",
+    "stylelint-processor-styled-components": "1.10.0",
```
Comment (milan-bc) [inline package.json:196]: things are upgraded to latest

File: .stylelintrc [hunk 3]
```diff
@@ -6,7 +6,8 @@
   ],
   "rules": {
     "property-no-vendor-prefix": null,
-    "media-feature-name-blacklist": ["max-width", "min-width"],
+    "media-feature-name-disallowed-list": ["max-width", "min-width"],
+    "at-rule-disallowed-list": ['media'],
```
Comment (milan-bc) [inline .stylelintrc:10]: no more usage of `media` allowed inside styled components 🔥

**Review by plondon — APPROVED** [review]

**Pull request merged**